build_hall.sh

```diff
@@ -443,6 +443,36 @@ gegede-cli duneggd/Config/WORLDggd.cfg \
            -w World -o SAND_opt3_STT1.gdml
 fi
 
+
+if [ $option = "all" -o $option = "only_sand_def_stt" ];
+then
+gegede-cli duneggd/Config/WORLDggd.cfg \
+           duneggd/Config/ND_Hall_Air_Volume_Only_SAND.cfg \
+           duneggd/Config/ND_Hall_Rock.cfg \
+           duneggd/Config/ND_ElevatorStruct.cfg \
+           duneggd/Config/SAND_MAGNET.cfg \
+           duneggd/Config/SAND_INNERVOLOPT_DefaultSTT.cfg \
+           duneggd/Config/SAND_ECAL.cfg \
+           duneggd/Config/SAND_STT/STT_Default.cfg \
```

Comment:
You have this "default geometry", but it is not actually used in the default that is built when no argument is given to "build_hall.sh". If this is the default, then it should be used for the "prod" options. Currently, that is still using "STT1.cfg", whatever that is.

Reply:
The default configuration, has some volume naming changes with some slight modifications. The sandreco package is sensitive to volume names. In order to not break the sandreco code due to volume name conflicts, this is not added as part of the prod geometries as we talked about sometime ago. Moreover after discussion we have decided to rename the argument in the build_hall.sh script as default -> complete, backup -> initial respectively.